browser/help/faq/technical-details/what-genetic-ancestry-groups-are-represented-in-the-gnomad-data.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { BaseTable, Link } from '@gnomad/ui'
+
+import { GNOMAD_POPULATION_NAMES } from '@gnomad/dataset-metadata/gnomadPopulations'
+import sampleCounts from '@gnomad/dataset-metadata/sampleCounts'
+
+import TableWrapper from '../../../src/TableWrapper'
+
+const SampleCountTable = styled(BaseTable)`
+  td {
+    text-align: right;
+  }
+
+  tbody tr:last-child {
+    th,
+    td {
+      border-bottom-color: #000;
+    }
+  }
+`
+
+const SampleCountTables = () => (
+  <div>
+    <div>
+      <h4>gnomAD v4</h4>
+      <p>
+        For current stats on genetic ancestry groups present in v4, please see the{' '}
+        {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
```

Comment:
General comment here, no action required, just venting: it's a bummer having to add new `@ts-expect-error`s, I'm looking forward to having the breathing room to work on overhauling GBTK again.